### PR TITLE
Add health check

### PR DESCRIPTION
**Pull request by drasko**
Signed-off-by: Drasko DRASKOVIC <drasko.draskovic@gmail.com>

### CODE REVIEW

File: service.go [hunk 1]
```diff
@@ -24,6 +24,9 @@ var (
 // Service specifies an API that must be fullfiled by the domain service
 // implementation, and all of its decorators (e.g. logging & metrics).
 type Service interface {
+	// Health check
+	HealthCheck() (string, error)
```
Comment (mijicd) [inline service.go]: This should not be the part of the domain. The data health checks could potentially return is usually bound to some "infrastructural" info. Hence, remove it from here.

File: health.go [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package manager
+
+// Health struct
+type Health struct {
```
Comment (mijicd) [inline health.go]: Unnecessary, the empty response with status 200 is more than enough for now.

File: api/transport.go [hunk 3]
```diff
@@ -122,6 +129,11 @@ func MakeHandler(svc manager.Service) http.Handler {
 	return r
 }
 
+func decodeHealth(_ context.Context, r *http.Request) (interface{}, error) {
+	req := healthReq{}
```
Comment (mijicd) [inline api/transport.go]: Inline this.

File: service.go (hunk 1 above)
Comment (drasko) [reply service.go]: So, I should not call `svc.HealthCheck()` from `endpoint.go`, but directly return OK from `healthEndpoint()`?

File: api/transport.go (hunk 3 above)
Comment (drasko) [reply api/transport.go]: You mean `return healtReq{}, nil`? I find it more readable this way, for people to learn and understand `go-kit`, that what you return from these `decodeFoo` functions is actually your `fooReq{}` that you should have defined in `resources.go`

**Comment by drasko** [comment]
Replaced by https://github.com/mainflux/manager/pull/6

**Pull request closed without merging**